### PR TITLE
[Documentation] Concepts: Orders Fixes

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes
| New docs?    | no
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/orders.rst [hunk 1]
```diff
@@ -35,6 +35,24 @@ Then get a channel to which you would like to add your Order. You can get it fro
...
+    $order->setLocale($locale);
+
+And a currency:
+
+.. code-block:: php
+
+    /** @var CurrencyInterface $currency */
+    $Currency = $this->container->get('sylius.context.currency')->getCurrency();
+
+    $order->setCurrency($currency);
```
Comment (michalmarcinkowski) [inline docs/book/orders.rst]: Since we add a currency, we should also add an exchange rate.

Comment (NeverResponse) [inline docs/book/orders.rst]: Moreover this is wrong. Order doesn't have a `currency` object anymore, it's split between `currencyCode` and `exchangeRate`. Same goes with `locale` => `localeCode`.
Also, both currency and locale contexts return code atm, so `->get*Code()`.

Comment (TheMadeleine) [inline docs/book/orders.rst]: fixed.

**Comment by pjedrzejewski** [comment]
Thank you Magda!

**Pull request merged**